src/OpenTelemetry.AutoInstrumentation/Configurations/ConfigurationKeys.cs

```diff
@@ -189,14 +194,14 @@ public static class Logs
         public const string IncludeFormattedMessage = "OTEL_DOTNET_AUTO_LOGS_INCLUDE_FORMATTED_MESSAGE";
 
         /// <summary>
-        /// Configuration key for comma separated list of enabled logs instrumentations.
+        /// Configuration key for enabling all log instrumentations.
```

Comment:
enabling or disabling? - traces and metrics has disabling and her enabling.

Reply:
Both are correct, but it is good to have same versions for all of them. Fixed aea539c6d22f5eec8be2d3e7b92b332828f4d128